woocommerce/storefront
# Implement stylelint and WordPress CSS coding standards

Rebased and updated https://github.com/woocommerce/storefront/pull/1431

Closes #1431

### How to test the changes in this Pull Request:

- Run npm run css. You should see no linting errors.
- Open a WordPress installation with StoreFront enabled and check for visual regressions.

### Changelog

> Tweak – update stylelint to use stylelint-config-wordpress\scss config and refactor SCSS files to follow WordPress CSS coding standards.
